### ISSUE TITLE
storage.tsdb.retention.size usage is not properly updated in document

### ISSUE THREAD

**Issue by venkatbvc** [prometheus/prometheus#7906]
**What did you do?**

In the https://prometheus.io/docs/prometheus/2.20/storage/ link, Following is mentioned regarding retention.size

--storage.tsdb.retention.size: [EXPERIMENTAL] This determines the maximum number of bytes that storage blocks can use (note that this does not include the WAL size, which can be substantial). The oldest data will be removed first. Defaults to 0 or disabled. This flag is experimental and can be changed in future releases. Units supported: B, KB, MB, GB, TB, PB, EB. Ex: "512MB" 

But where in Release notes of 2.15.0, see that it is mentioned that retention.size includes wal also. 
[ENHANCEMENT] TSDB: WAL size is now used for size based retention calculation. #5886 

Does documentation needs correction? 

**What did you expect to see?**
Documentation should mention that WAL size is also considered by this flag.
**What did you see instead? Under which circumstances?**

**Environment**
Linux
* System information:

	insert output of `uname -srm` here

* Prometheus version:

	insert output of `prometheus --version` here
2.20

* Alertmanager version:

	insert output of `alertmanager --version` here (if relevant to the issue)

* Prometheus configuration file:
```
insert configuration here
```

* Alertmanager configuration file:
```
insert configuration here (if relevant to the issue)
```

* Logs:
```
insert Prometheus and Alertmanager logs relevant to the issue here
```
https://groups.google.com/g/prometheus-users/c/wgwcjdieBu0/m/YJeq6stUAQAJ

**Comment by brian-brazil** [comment]
Would you like to send a PR?

**Comment by venkatbvc** [comment]
@brian-brazil  yes.

**Comment by nrchakradhar** [comment]
Fixed by #7942
